dev/grid.html

```diff
@@ -34,6 +34,16 @@
       tooltip.generator = ({ column, item }) => {
         return column && column.path && item ? `Tooltip ${column.path} ${item.name}` : '';
       };
+
```

Comment:
Please only add the attribute to the `<vaadin-grid-selection-column>` element on the dev page. No need for the toggle controls.

Reply:
Done